### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by masihtak** [lusaxweb/vuesax-next#34]
I noticed some lack of accessibility in some of your components which I will walk through:

## Labels

Unfortunately in default input fields and most of the input fields , you only using a placeholder as a label! (I DO know you have hidden labels for the screen readers but the problem is you are sacrificing a11y for the beauty! )
Placeholders are appealing because of their minimal, space-saving aesthetic. This is because placeholder text is placed inside the field. But this is a problematic way to give users a hint. First, they disappear when the user types. Disappearing text is hard to remember, which can [cause errors](https://www.uxmatters.com/mt/archives/2010/03/dont-put-hints-inside-text-boxes-in-web-forms.php) if, for example, the user forgets to satisfy one of the password rules. Users often mistake placeholder text for a value, causing the field to be skipped.
Gray-on-white text lacks sufficient contrast, making it generally hard-to-read. And to top it off, some browsers don’t support placeholders. [How the Web Became Unreadable](https://medium.com/backchannel/how-the-web-became-unreadable-a781ddc711b6).

We already have label input you can use them! Yes I know but the fact is frameworks are meant to be standard and by doing this you're making bad habits for developers who use your framework those who want to use your framework for design and focus on the back-end, so you speared this problem like poison in the web!

## Float Labels

I can see you also using some float labels `label-placeholder` . Unfortunately, there are several problems with this approach.

First, there is no space for a hint because the label and hint are one and the same. Second, they’re hard to read, due to their poor contrast and small text, as they’re typically designed. Third, like placeholders, they may be mistaken for a value and could get cropped.

> “Seems like a lot of effort when you could simply put labels above inputs & get all the benefits/none of the issues.”

Quirky and minimalist interfaces don’t make users feel awesome — obvious, inclusive, and robust interfaces do.

## Size

[Size matters!](https://medium.com/@zacdicko/size-matters-accessibility-and-touch-targets-56e942adc0cc) Your medium input size is about 34px which is not good.

## Borders

A text box should look like a text box. Empty boxes signify “fill me in” by virtue of being empty, like a coloring-in book. This also means that the empty space should be boxed in (bordered). Removing the border, or having only a bottom border ( like what you have [here](https://vuesax.com/docs/components/Input.html#border)) A bottom border might at first appear to be a separator. Even if you know you have to fill something in, does the value go above the line or below it?

## Focus styles
#8

If you agree with these principles I'd be glad to start working on them and make Vuesax even much better!
